src/main/webapp/app/entities/modeling-exercise/modeling-exercise-update.component.ts

```diff
@@ -42,16 +39,31 @@ export class ModelingExerciseDialogComponent implements OnInit {
     domainCommandsGradingInstructions = [new KatexCommand()];
 
     constructor(
-        public activeModal: NgbActiveModal,
         private jhiAlertService: JhiAlertService,
         private modelingExerciseService: ModelingExerciseService,
         private courseService: CourseService,
         private exerciseService: ExerciseService,
         private eventManager: JhiEventManager,
         private exampleSubmissionService: ExampleSubmissionService,
+        private activatedRoute: ActivatedRoute,
+        private router: Router,
     ) {}
 
     ngOnInit() {
+        // This is used to scroll page to the top of the page, because the routing keeps the position for the
+        // new page from previous page.
+        window.scroll(0, 0);
```

Comment:
Same suggestion as in the other PR: It would be nice if you use the WindoRef from the window.service.ts here

Reply:
Same solution!